src/Collection/ProcessArgumentsCollection.php

```diff
@@ -150,4 +150,22 @@ public function addArgumentWithCommaSeparatedFiles($argument, FilesCollection $f
 
         $this->add(sprintf($argument, implode(',', $paths)));
     }
+
+    /**
+     * @return array|string
+     */
+    public function generateCliCommand()
+    {
+        $commandline = $this->getValues();
+
+        // Backwards compatibility layer for Symfony Process < 3.3
+        if (class_exists('Symfony\Component\Process\ProcessBuilder')) {
```

Comment:
Isn't it better to always return a string and escape the arguments manually?
All Symfony versions can handle escaped strings, so this would make the API easier.